<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>run_simulation_window</class>
 <widget class="QDialog" name="run_simulation_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="sequentialRdn">
      <property name="text">
       <string>Sequential</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="optimisticRdn">
      <property name="text">
       <string>Optimistic</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="conservativeRdn">
      <property name="text">
       <string>Conservative</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of processes:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinProcess">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>591</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ROSS BUILDER FOLDER</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="rossbuilderBtn">
      <property name="text">
       <string>file..</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Model to simulate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="modelBtn">
      <property name="text">
       <string>file</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Routes file:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="routesBtn">
      <property name="text">
       <string>file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
